### ISSUE TITLE
Inso CLI Deprecated packages

### ISSUE THREAD

**Issue by ppazos** [getinsomnia/insomnia#4247]
### Expected Behavior

There shouldn't be deprecated packages when installing.

### Actual Behavior

```shell
$ npm install --global insomnia-inso
npm WARN deprecated @stoplight/spectral@5.9.2: Spectral's latest version is now available as @stoplight/spectral-cli on npm
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/boom@7.4.11: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/cryptiles@4.2.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/b64@4.2.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/bounce@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/teamwork@3.3.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated mkdirp-promise@1.1.0: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated node-pre-gyp@0.15.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN checkPermissions Missing write access to /usr/lib/node_modules

```

### Reproduction Steps

$ npm install --global insomnia-inso

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

This is under Linux Mint 19.3

### Insomnia Version

2021.3.0

### What operating system are you using?

Other Linux

### Operating System Version

Mint 19.3

### Installation method

$ npm install --global insomnia-inso

### Last Known Working Insomnia version

not sure

**Comment by filfreire** [comment]
Hi @ppazos, thank you for reporting this!

I can still reproduce this with latest Inso CLI (3.0.0), though with slightly different deprecation warnings:

```
npm WARN deprecated mkdirp-promise@1.1.0: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
npm WARN deprecated @hapi/teamwork@4.0.0: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @stoplight/spectral@5.9.2: Spectral's latest version is now available as @stoplight/spectral-cli on npm
```

We'll update you here if there's any changes or fixes at some point to these.

Feel free to report any other issues you might come across 👍

**Comment by filfreire** [comment]
Still reproducible in version `3.3.0`:
```
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/teamwork@4.0.0: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated mkdirp-promise@1.1.0: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
```

**Comment by filfreire** [comment]
Inso is no longer avaialble through npmjs, closing this one for now.
